scripts/build-config-fixtures.js

```diff
@@ -0,0 +1,52 @@
+import yazl from 'yazl'
+import fs from 'node:fs/promises'
+import { createWriteStream } from 'node:fs'
+
+const CONFIG_FIXTURES_PATH = new URL(
+  '../tests/fixtures/config',
+  import.meta.url
+).pathname
+const dir = await fs.readdir(CONFIG_FIXTURES_PATH)
+console.log('zipping config fixtures')
+for (const fileOrFolder of dir) {
+  const p = `${CONFIG_FIXTURES_PATH}/${fileOrFolder}`
+  const stat = await fs.stat(p)
+  if (stat.isDirectory()) {
+    zipFolder(p)
+  }
+}
+
+/**
+ * @param {String} path
+ */
+async function zipFolder(path) {
+  const zip = new yazl.ZipFile()
+  const zipPath = `${path}.zip`
+  const wstream = createWriteStream(zipPath)
+  zip.outputStream.pipe(wstream).on('close', function () {
+    console.log(`done zipping ${zipPath.split('/').at(-1)}`)
+  })
+  const contents = await fs.readdir(path)
+  for (const content of contents) {
+    const filePath = `${path}/${content}`
+    for await (const p of walk(filePath)) {
+      const zipPath = p.replace(`${path}/`, '')
+      zip.addFile(p, zipPath)
+    }
+  }
+  zip.end()
+}
+
+/**
+ */
+async function* walk(path) {
+  const stat = await fs.stat(path)
+  if (stat.isDirectory()) {
+    const dir = await fs.readdir(path)
+    for (const newPath of dir) {
+      yield* await walk(`${path}/${newPath}`)
```

Comment:
Two nitpicks:

1. `await` has no effect here and can be removed.
2. `/` might not work on Windows. Could we use `path.join` instead?

```suggestion
      yield* walk(join(path, newPath))
```